packages/foreman/foreman-fapolicyd/foreman-fapolicyd.spec

```diff
@@ -0,0 +1,89 @@
+Name:     foreman-fapolicyd
+Version:  1.0.0
+Release:  1%{?dist}
+Summary:  Foreman fapolicyd rules
+
+Group:    System Environment/Base
+License:  GPLv3+
+URL:      https://theforeman.org
+Source0:  https://codeload.github.com/theforeman/%{name}/tar.gz/v%{version}#/%{name}-%{version}.tar.gz
+
+BuildArch: noarch
+
+Requires: fapolicyd
+Requires: foreman-proxy-fapolicyd
+
+%description
+Foreman fapolicyd rules
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_sysconfdir}/fapolicyd/rules.d
+cp 60-foreman.rules %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+cp 61-foreman-proxy.rules %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/61-foreman-proxy.rules
+
+%post
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%postun
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%files
+%defattr(-,root,root)
+%attr(0644,root,fapolicyd) %{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+
+%package -n foreman-proxy-fapolicyd
+Summary: Foreman Proxy fapolicyd rules
+Group:   System Environment/Base
+
+Requires: fapolicyd
```

Comment:
I think you also need a `Requires(postun): fapolicyd` as you're using `fagenrules` in `%postun`